baseplate/lib/thrift_pool.py

```diff
@@ -142,6 +142,9 @@ class ThriftConnectionPool:
 
     """
 
+    class _PoolCheckoutException(Exception):
```

Comment:
Can this be at the module level so we don't have to access it with cls/self?

Reply:
I removed it altogether